.github/actions/setup/action.yml

```diff
@@ -51,7 +51,7 @@ runs:
         set -e
         sudo xcode-select --switch /Applications/Xcode_15.4.app
         brew update
-        brew install autoconf autoconf-archive automake coreutils bash ffmpeg ninja wabt ccache unzip qt llvm@18
+        brew install autoconf autoconf-archive automake coreutils bash nasm ninja wabt ccache unzip qt llvm@18
```

Comment:
surely we don't actually need nasm for aarch64 builds? 🤔

Reply:
I added it because I got the following error without it:
```
CMake Error at scripts/cmake/vcpkg_find_acquire_program.cmake:166 (message):
  Could not find nasm.  Please install it via your package manager:

      brew install nasm
```